### DISCUSSION TITLE
Quieren que nos reunamos hoy?

### DISCUSSION THREAD

**Discussion by sebasvelasco353** [sebasvelasco353/FDP-2022_01#197] [Announcements]
Hola guys, como les había dicho si quieren podemos vernos hoy para resolver dudas, pero necesito que uds me confirmen si van a ir... Me responden por acá porfa.

Gracias.

**Comment by ghost** [comment]
Profe, le confirmo que por el momento no tengo ninguna duda, pero de igual forma ire
